cloud_blobstore/s3.py

```diff
@@ -149,11 +149,18 @@ def upload_file_handle(
             self,
             bucket: str,
             key: str,
-            src_file_handle: typing.BinaryIO):
+            src_file_handle: typing.BinaryIO,
+            content_type: str = None,
+            metadata: dict = None):
+        extra_args = None
+        if content_type or metadata:
+            extra_args.update({'ContentType': content_type} if content_type else {})
```

Comment:
Can you call `.update` on `None`?

Reply:
I was try to avoid creating that extra dict, but I guess i'll have to. Good catch. Will fix.